src/site/_includes/partials/post.njk

```diff
@@ -14,6 +14,8 @@
   {# Make sure Actions are first in the tab order. #}
   <web-actions actions="share|subscribe" authors="{{ authors | expandContributors | join('|') }}"></web-actions>
 
+  {% include 'partials/banner.njk' %}
```

Comment:
Are we putting the banner literally everywhere now?

Reply:
mmmm yeah. I was going to try it and see if folks kill me.

Reply:
also wanted to mention that it seems like /web puts it pretty much everywhere as well (at least, it's on everything /updates that I saw).